lsp-iedit.el

```diff
@@ -0,0 +1,55 @@
+;;; lsp-iedit.el --- `iedit' integration -*- lexical-binding: t -*-
```

Comment:
Could you follow other files template?
https://github.com/emacs-lsp/lsp-mode/blob/master/lsp-headerline.el#L3-L16

Reply:
Should this by `Copyright lsp-maintainers` or `Copyright <me>`?

Reply:
lsp-maintainers

Reply:
Fixed, with `Copyright emacs-lsp maintainers`.